Subject: Revert "Merge 104421 - Fix library paths for preloading NSS on Ubuntu 11.10."

This backs out the change introduced by r104798, which allowed NSS to
actually be initialized, where "initialized" means "trip a sandboxing
bug and take down the renderer".

Works around: http://bugs.debian.org/647992

Index: sid/src/crypto/nss_util.cc
===================================================================
--- sid.orig/src/crypto/nss_util.cc	2012-05-16 15:50:13.258129408 +0200
+++ sid/src/crypto/nss_util.cc	2012-05-17 15:01:26.470355730 +0200
@@ -35,7 +35,6 @@
 #include "base/scoped_temp_dir.h"
 #include "base/stringprintf.h"
 #include "base/threading/thread_restrictions.h"
-#include "build/build_config.h"
 #include "crypto/scoped_nss_types.h"
 
 #if defined(OS_CHROMEOS)
@@ -723,18 +722,9 @@
   // Use relative path to Search PATH for the library files.
   paths.push_back(FilePath());
 
-  // For Debian derivatives NSS libraries are located here.
+  // For Debian derivaties NSS libraries are located here.
   paths.push_back(FilePath("/usr/lib/nss"));
 
-  // Ubuntu 11.10 (Oneiric) places the libraries here.
-#if defined(ARCH_CPU_X86_64)
-  paths.push_back(FilePath("/usr/lib/x86_64-linux-gnu/nss"));
-#elif defined(ARCH_CPU_X86)
-  paths.push_back(FilePath("/usr/lib/i386-linux-gnu/nss"));
-#elif defined(ARCH_CPU_ARMEL)
-  paths.push_back(FilePath("/usr/lib/arm-linux-gnueabi/nss"));
-#endif
-
   // A list of library files to load.
   std::vector<std::string> libs;
   libs.push_back("libsoftokn3.so");
@@ -757,7 +747,7 @@
   if (loaded == libs.size()) {
     VLOG(3) << "NSS libraries loaded.";
   } else {
-    LOG(ERROR) << "Failed to load NSS libraries.";
+    LOG(WARNING) << "Failed to load NSS libraries.";
   }
 #endif
 }
